ge/org.osate.ge/src/org/osate/ge/internal/ui/editor/AgeContentOutlinePage.java

```diff
@@ -447,6 +451,9 @@ private void updateOutlineSelectionIfLinked() {
 				if (treeViewer != null && treeViewer.getContentProvider() != null
 						&& !outlineNodes.equals(editor.getSelectedDiagramElementSet())) {
 					treeViewer.setSelection(buildDiagramNodeTreeSelectionFromEditor());
+					if (revealNode) {
+						editor.reveal(outlineNodes.iterator().next());
```

Comment:
I don't think `revealNode` and the call to `reveal` is desired. This is called to update the outline to match the editor. I don't think we want to reveal the current selection when the link with editor button is selected.

Reply:
Yep, you are correct... not sure why I did that.  fixed